.github/workflows/zero-downtime-tests.yml

```diff
@@ -47,6 +47,7 @@ jobs:
       KUBECONFIG: ${{ matrix.KUBECONFIG }}
       KEPTN_NAMESPACE: "keptn-zd-test-${{ github.run_number }}-${{ github.run_attempt }}"
       GO_VERSION: 1.17
+      GOMAXPROCS: 10
```

Comment:
i think you could also move this variable to the step where it is needed (`test_zero_downtime`)

Reply:
I tought to keep the go env setup togheter, but sure I can move it